CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ FetchContent_Declare(
         GIT_TAG        v2.13.9 # or a later release
 )
 
-FetchContent_MakeAvailable(Catch2)
```

Comment:
why did you remove the Catch2 lib?

Reply:
Catch2 and the file format project were temporarily removed because they require Python and I couldn't be bothered installing it.